requirements.txt

```
@@ -7,7 +7,8 @@ h11 @ git+https://github.com/python-hyper/h11.git@master
 # Explicit optionals
 a2wsgi==1.10.7
 wsproto==1.2.0
-websockets==13.1
+websockets==13.1; python_version < '3.9'
+websockets==14.1; python_version >= '3.9'
```

Let's bump this in another PR.
```suggestion
websockets==13.1
```